src/format.js

```diff
@@ -24,5 +24,31 @@ module.exports = {
         if (m >= 10000) return (m / 1000).toFixed(1) + ' km';
         if (m >= 100) return (m / 1000).toFixed(2) + ' km';
         return m.toFixed(0) + ' m';
-    }
+    },
+
+    en: function(m) {
+        return m;
+    },
+
+    pt: function(m) {
+        m = m.replace(/\bNorth/g, "Norte");
+        m = m.replace(/\bnorth/g, "Norte");
+        m = m.replace(/\bSouth/g, "Sul");
+        m = m.replace(/\bsouth/g, "Sul");
+        m = m.replace(/\bEast/g, "Este");
+        m = m.replace(/\bWest/g, "Oeste");
+        m = m.replace(/\bonto/g, "para");
+        m = m.replace(/\bon/g, "na");
+        m = m.replace(/\bHead/g, "Siga para");
+        m = m.replace(/Turn right/g, "Vire à direita");
+        m = m.replace(/Turn left/g, "Vire à esquerda");
+        m = m.replace(/\bContinue/g, "Continue");
+        m = m.replace(/You have arrived at your destination/g, "Chegou ao seu destino");
+/*
```

Comment:
yes, and I still need to add a couple of instructions, but is this the way to do it?